packages/eslint-config-algolia/flowtype.js

```diff
@@ -1,5 +1,6 @@
 // eslint-disable-next-line import/no-commonjs
 module.exports = {
-  extends: ['plugin:flowtype/recommended', './base.js', 'prettier/flowtype'],
+  extends: ['plugin:flowtype/recommended'],
   plugins: ['flowtype'],
+  parser: 'babel-eslint',
```

Comment:
let's remove this config in a next PR/manor

Reply:
you mean the entire file flowtype.js ?

Reply:
yes, I don't think we use flow anywhere anymore